Assets/Group1/Scripts/Player/PlayerMovement.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerMovement : MonoBehaviour
+{
+    [SerializeField] private float _speed;
+
+    private Vector2 _direction;
+
+    private void Update()
+    {
+        Move();
+    }
+
+    private void Move()
+    {
+        float horizontalMovement = Input.GetAxisRaw("Horizontal");
+        float verticalMovement = Input.GetAxisRaw("Vertical");
```

Comment:
https://trello.com/c/UAeMGORw/17-%D0%BE%D1%82%D0%BA%D0%B0%D0%B7-%D0%BE%D1%82-%D1%81%D1%82%D1%80%D0%BE%D0%BA%D0%BE%D0%B2%D1%8B%D1%85-%D0%BB%D0%B8%D1%82%D0%B5%D1%80%D0%B5%D0%BB%D0%BE%D0%B2